dask_expr/_reductions.py

```diff
@@ -1259,6 +1259,41 @@ def _nlargest_slow(df, columns, n):
     return df.sort_values(by=columns).tail(n)
 
 
+def _nfirst(df, columns, n, ascending):
+    return df.sort_values(by=columns, ascending=ascending).head(n)
+
+
+def _nlast(df, columns, n, ascending):
+    return df.sort_values(by=columns, ascending=ascending).tail(n)
+
+
+class NFirst(NLargest):
```

Comment:
I don't like having `NFirst`, `NLast`, `NLargestSlow` and `NSmallestSlow` that all do basically the same. I also don't like that current simplification hierarchy. We should refactor this in a follow-up.